util/util.go

```diff
@@ -163,13 +162,17 @@ func AddImageNames(store storage.Store, image *storage.Image, addNames []string)
 func GetFailureCause(err, defaultError error) error {
 	switch nErr := errors.Cause(err).(type) {
 	case errcode.Errors:
-		return cli.NewMultiError([]error(nErr)...)
-	case errcode.Error, *url.Error:
-		return nErr
-	default:
 		// HACK: In case the error contains "not authorized" like in
 		//       https://github.com/containers/image/blob/master/docker/docker_image_dest.go#L193-L205
 		// TODO(bshuster): change "containers/images" to return "errcode" rather than "error".
```

Comment:
I think we can remove these two comments:
```
// HACK: ....
// TODO(bshuster): ...
```

Reply:
Zapped, thanks!